opae-sdk-rpm.spec

```diff
@@ -0,0 +1,231 @@
+Summary:        Open Programmable Acceleration Engine
+Name:           opae
+Version:        1.4.0
```

Comment:
1.4.1

Reply:
Currently we are at 1.4.0, will update to 1.4.1 when we do the next release. :)